src/core/reference/include/ngraph/runtime/reference/mvn.hpp

```diff
@@ -86,6 +86,25 @@ void mvn_6(const T* arg,
         divide(out, tmp_buffer.data(), out, in_shape, reduced_shape, op::AutoBroadcastType::NUMPY, true);
     }
 }
+
+template <typename T>
+AxisSet mvn_6_reduction_axes(const HostTensorPtr& axes_input, size_t rank) {
```

Comment:
Please use ov::Tensor instead of ngraph::HostTensoe

Reply:
Done.